zcash_client_backend/CHANGELOG.md

```diff
@@ -52,13 +52,13 @@ and this library adheres to Rust's notion of
   - `wallet::input_selection::ShieldingSelector` has been
     factored out from the `InputSelector` trait to separate out transparent
     functionality and move it behind the `transparent-inputs` feature flag.
-  - `TransparentAddressMetadata` (which replaces `zcash_keys::address::AddressMetadata`).
 - `zcash_client_backend::fees::{standard, sapling}`
 - `zcash_client_backend::fees::ChangeValue::new`
 - `zcash_client_backend::wallet`:
   - `Note`
   - `ReceivedNote`
   - `WalletSaplingOutput::recipient_key_scope`
+  - `wallet::TransparentAddressMetadata` (which replaces `zcash_keys::address::AddressMetadata`).
```

Comment:
You added `wallet::`, but it's under a `::wallet` parent bullet, and is different from the rest in this list.